tools/migrate/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"context"
+	"log/slog"
+	"os"
+
+	"github.com/openmeterio/openmeter/internal/ent/db"
+	"github.com/openmeterio/openmeter/pkg/framework/entutils"
+)
+
+func main() {
+	PG_URL := os.Getenv("POSTGRES_URL")
+	if PG_URL == "" {
+		PG_URL = "postgres://postgres:postgres@localhost:5432/postgres"
+	}
+
+	logger := slog.New(slog.NewJSONHandler(os.Stdout, &slog.HandlerOptions{Level: slog.LevelDebug}))
+	slog.SetDefault(logger)
```

Comment:
Do you know if ent uses a global logger instance?

Reply:
I don't think it logs anything by itself, only if you add logging through a custom hook. `slog.SetDefault(logger)` is just there for good measure, but yes, ineffective as is right now.